runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -11014,20 +11014,19 @@ static TR::Register* inlineFindElementFromArray(TR::Node* node, TR::CodeGenerato
    uint8_t shift = 0;
    uint8_t* shuffleMask = NULL;
    auto compareOp = BADIA32Op;
-   switch(node->getSymbol()->castToMethodSymbol()->getMethod()->parmType(1))
+   if(isCompressed)
       {
       case TR::Int8:
          shuffleMask = MASKOFSIZEONE;
          compareOp = PCMPEQBRegReg;
          shift = 0;
-         break;
+      }
+   else
+      {
       case TR::Int16:
```

Comment:
The `case` statement should also be removed.

Reply:
6a8cc98